interfaces/console/Dockerfile

```diff
@@ -1,62 +1,27 @@
-# FROM nginx
-# COPY  build /usr/share/nginx/html
-# RUN rm /etc/nginx/conf.d/default.conf
-# COPY nginx/nginx.conf /etc/nginx/conf.d
-# EXPOSE 8080
-# CMD ["nginx", "-g", "daemon off;"]
+FROM node:18-alpine AS base
 
-# FROM node:18-alpine AS deps
-# RUN apk add --no-cache libc6-compat
-# WORKDIR /
-
-# COPY package.json yarn.lock ./
-# RUN  yarn install --production
-
-# FROM node:18-alpine AS builder
-# WORKDIR /
-# COPY --from=deps /node_modules ./node_modules
-# COPY . .
-
-# ENV NEXT_TELEMETRY_DISABLED 1
-
-# RUN yarn build
-
-# FROM node:18-alpine AS runner
-# WORKDIR /
-
-# ENV NODE_ENV production
-# ENV NEXT_TELEMETRY_DISABLED 1
-
-# RUN addgroup --system --gid 1001 nodejs
-# RUN adduser --system --uid 1001 nextjs
-
-# COPY --from=builder --chown=nextjs:nodejs /.next ./.next
-# COPY --from=builder /node_modules ./node_modules
-# COPY --from=builder /package.json ./package.json
-
-# USER nextjs
-
-# EXPOSE 3000
-
-# ENV PORT 3000
+FROM base AS deps
+RUN apk add --no-cache libc6-compat
+WORKDIR /app
 
-# CMD ["yarn", "start"]
+COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* ./
+RUN \
```

Comment:
Can you use multi stage build and maybe combine the run commands:
I think you can do something like this:
```
# Build stage
FROM node:alpine AS build
WORKDIR /app
COPY package*.json ./
RUN npm ci --only=production
COPY . .
RUN npm run build

# Production stage
FROM node:alpine
WORKDIR /app
COPY --from=build /app/package*.json ./
COPY --from=build /app/.next ./.next
COPY --from=build /app/public ./public
ENV NODE_ENV=production
CMD ["npm", "start"]
```